bin/models.py

```diff
@@ -7,8 +7,6 @@
 
 if config.REDIS_ENABLED:
     database = Redis(host=config.REDIS_HOST, port=config.REDIS_PORT)
-else:
-    raise EnvironmentError("In-memory database is not supported for now")
```

Comment:
Well, maybe one last modification, shall we not add a warning?

Reply:
Un message d'avertissement dans les tests unitaires doit être considéré comme un erreur. Étant donné l'endroit où on se connecte à redis (contexte global lorsque le module est importé) on aura beaucoup de mal à passer l'avertissement sous silence, rediriger stdout/stderr/logging/warnings me semble risqué et on devra rajouter du code pour parer aux éventualités.

La structure est pas dingue mais vu qu'on a très peu de code je pense qu'une autre architecture (plus test-proof) serait trop compliqué étant donné la taille de la codebase

Reply:
Ou bien on vire définitivement le `--no-redis` et on dit que le service *DOIT* pouvoir se connecter à une base de donnée pour démarrer

Reply:
Étant donné que le service ne fonctionne pas sans, l'enlever, même temporairement, me semble mieux.

Reply:
Ok, on peut faire ça, tu t'en occupes ? :)

Reply:
Yep, si tu vois pas de commit avant ce soir c'est que j'ai oublié 🙃